packages/web-app-files/src/views/spaces/Project.vue

```diff
@@ -0,0 +1,396 @@
+<template>
+  <div class="space-overview">
+    <list-loader v-if="loadSpaceTask.isRunning" />
+    <template v-else>
+      <not-found-message v-if="!space.id" class="space-not-found oc-height-1-1" />
+      <div
+        v-else-if="isSpaceRoot"
+        class="oc-grid oc-grid-match oc-px-m oc-mt-m"
+        :class="{ 'oc-child-width-1-1@s': imageExpanded, 'oc-child-width-1-3@s': !imageExpanded }"
+      >
+        <div v-if="imageContent">
+          <img
+            :class="{ expanded: imageExpanded }"
+            class="space-overview-image oc-cursor-pointer"
+            alt=""
+            :src="'data:image/jpeg;base64,' + imageContent"
+            @click="toggleImageExpanded"
+          />
+        </div>
+        <div>
+          <h3 class="space-overview-name oc-mb-s">{{ space.name }}</h3>
```

Comment:
Not sure if there's a `<h2>` above in the DOM, if not this should be one instead

Reply:
There is not even a <h1>, so if change it to a h2 the style looks quite off...

Reply:
Correct tag usage and styling are two different matters, though :) let's discuss this in a call later, but there should never be a "smaller" heading if the previous size is not present (since the screenreader uses them for reasoning about the page content)

Reply:
bootstrap has h classes like h1,h2,h3 and so on, so you can manipulate the styling, we might want to add those, like
oc-h1 and so on...

Reply:
Did you discuss it? The `h1` is always supposed to be the page title, in our case we have it always visually hidden (but accessible for assistive technology). If that's not the case for the Spaces views, yet, we'd need to add it. Styling is indeed a different story.

Reply:
Made it `h1` for now. I think eventually we have to make a separate PR with a11y adjustments.